packages/@expo/config-plugins/src/ios/__tests__/DevelopmentTeam-test.ts

```diff
@@ -0,0 +1,70 @@
+import { ExpoConfig } from '@expo/config-types';
+import { fs as memfs, vol } from 'memfs';
+import path from 'path';
+
+import rnFixture from '../../plugins/__tests__/fixtures/react-native-project';
+import { getDevelopmentTeam, setDevelopmentTeamForPbxproj } from '../DevelopmentTeam';
+
+const fs = jest.requireActual('fs') as typeof import('fs');
+const baseExpoConfig: ExpoConfig = {
+  name: 'testproject',
+  slug: 'testproject',
+  platforms: ['ios'],
+  version: '1.0.0',
+};
+
+jest.mock('fs');
+jest.mock('node:fs', () => require('memfs').fs);
```

Comment:
Jest being jest here, unfortunately, it doesn't pick up our global `fs` mock when using `node:fs`.